### ISSUE TITLE
Kroger/Fred Meyer blocked again

### ISSUE THREAD

**Issue by jxchong** [GUI/covid-vaccine-spotter#146]
Akamai is back (yay... not)

**Comment by GUI** [comment]
😞 Yeah, looks like they've re-added session and captcha requirements to some of their endpoints. This is unfortunate, since I thought it was maybe purposeful when they opened their endpoints up a bit and no longer required session cookies, which is what finally allowed me to start scanning them nation-wide. I'll have to investigate more, but if this means I have to go back to the crazy system of moving mouse cursors around and solving captchas, then it may not be practical to scan again (since it would be so slow and costly).

**Comment by jxchong** [comment]
Yeah we were hoping it was purposeful too since Albertsons opened up in the same week.  :(

**Comment by petegordon** [comment]
Thanks! We were using your data and had retired our scraper; but have turned it back on.  We could maybe expand this to be nationwide; but the processing takes a lot of time to do it (a zipcode every few minutes)--started with the zipcodes that Kroger exist in for the state of Ohio and then update about every ~8 hours for all locations in the state.   The JSON data updating live is here... https://github.com/petegordon/VaccinateOhio_KrogerScraper/tree/main/ZipLocationsVaccines

**Comment by GUI** [comment]
Kroger may be somewhat back in action. I've been working with the Kroger team to get access (thanks, Kroger!). I'm still ironing out some access kinks, so it may periodically stop updating again overnight, but I'm hoping I can get all of this sorted with them tomorrow. 🤞
